src/aat/resources/features/F-108/S-603.td.json

```diff
@@ -0,0 +1,87 @@
+{
+  "_guid_": "S-603",
+  "_extends_": "F-103_Test_Data_Base",
+  "specs": [
+    "a role with read access to the case state",
+    "is configured to search for the previously created case via exact match",
+    "contains the previously created case data"
+  ],
+
+  "request": {
+    "body": {
+      "query": {
+        "match": {
+          "reference": "${[scenarioContext][childContexts][Private_Case_Creation_Autotest1_Data][testData][actualResponse][body][id]}"
+        }
+      }
+    }
+  },
+
+
+  "expectedResponse": {
```

Comment:
a lot of response details seems to be re-usable from a common response object to be extended. That would make these response body details better maintainable.

Reply:
pls check if applicable. @tomxelliott